src/libgriddyn/griddyn/bits/griddyn_structs.h

```diff
@@ -0,0 +1,18 @@
+#ifndef LIBGRIDDYN_BITS_GRIDDYN_STRUCTS_H
+#define LIBGRIDDYN_BITS_GRIDDYN_STRUCTS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct griddyn_sim_ { char _; };
+struct griddyn_bus_ { char _; };
+struct griddyn_link_ { char _; };
+struct griddyn_load_ { char _; };
+struct griddyn_generator_ { char _; };
```

Comment:
What exactly are you doing here?

Reply:
The default struct size for C is zero, but for C++ is one. This standardizes them in case anyone actually creates these structs on the C side for use on the C++ side. 

Unfortunately, they do add size to the C++ objects that inherit from them, because EBO no longer happens. I think an alternative option is just declaring them on the C side, and defining them to be empty on the C++ side? I need to try that, anyway.

Reply:
Fixed now, they're declared in C and defined in C++. There shouldn't be name-mangling issues, since everything that uses these structs uses them as pointers, from an `extern "C"` context.